types/config.d.ts

```diff
@@ -146,6 +146,8 @@ declare global {
         id: string;
         weight: number;
         group?: string;
+        relevantAudits?: string[];
```

Comment:
minor nit: swap order of these two new fields to match prototype field ids (and in below file)